--- origsrc/Luach/Luach.pro	2010-02-20 12:30:16.000000000 -0600
+++ src/Luach/Luach.pro	2010-06-06 23:15:56.815606800 -0500
@@ -31,8 +31,7 @@ win32 {
 RESOURCES += Luach.qrc
 
 # Install binaries
-binary.path = /usr/bin/
-binary.files = luach
+target.path = /usr/bin/
 
 # Install ZmanimCLI
 zmanim.path = /usr/share/Luach/
@@ -45,7 +44,7 @@ icon.files = Icons/calendar.png
 # Install shortcut
 menu.path = /usr/share/applications
 menu.files = Luach.desktop
-INSTALLS += binary
+INSTALLS += target
 INSTALLS += zmanim
 INSTALLS += icon
 INSTALLS += menu
