--- origsrc/Luach/changelocation.ui	2009-10-12 13:36:42.000000000 -0500
+++ src/Luach/changelocation.ui	2010-06-07 03:40:06.693168000 -0500
@@ -273,6 +273,10 @@
        <property name="text">
         <string>ביטול</string>
        </property>
+       <property name="icon">
+        <iconset>
+         <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-cancel-16.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-cancel-16.png</iconset>
+       </property>
       </widget>
      </item>
      <item>
@@ -296,6 +300,10 @@
        <property name="text">
         <string>שמור</string>
        </property>
+       <property name="icon">
+        <iconset>
+         <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-ok-16.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-ok-16.png</iconset>
+       </property>
       </widget>
      </item>
     </layout>

