--- origsrc/luach/mainwindow.cpp	2010-06-20 02:05:39.000000000 -0500
+++ src/luach/mainwindow.cpp	2010-06-20 02:12:28.882352100 -0500
@@ -89,7 +89,7 @@ bool ShowZmanim;
 QString LANG;
 
 
-QString ZMANIMCLIPATH = "ZmanimCLI.jar";
+QString ZMANIMCLIPATH = "ZmanimCLI.jar:/usr/share/java/zmanim.jar";
 
 
 MainWindow::MainWindow(QWidget *parent)
@@ -118,7 +118,7 @@ MainWindow::MainWindow(QWidget *parent)
             toggleZmanimPanel(false);
         }
 
-        ZMANIMCLIPATH = "/usr/share/Luach/ZmanimCLI.jar";
+        ZMANIMCLIPATH = "/usr/share/Luach/ZmanimCLI.jar:/usr/share/java/zmanim.jar";
     }
 
     setWindowIcon(QIcon(":/Icons/calendar.png"));
@@ -452,7 +452,7 @@ void MainWindow::updateLabels(mHdate *da
 
     QString dstr = stringify(date->get_gyear()) + "/" + stringify(date->get_gmonth()) + "/" + stringify(date->get_gday());
 
-    args << "-jar" << ZMANIMCLIPATH << "-d" << dstr << "-lat" << stringify(latitude) << "-lon" << stringify(longitude) << "-e" << stringify(elevation) << "-tz" << TimeZone;
+    args << "-cp" << ZMANIMCLIPATH << "ZmanimCLI" << "-d" << dstr << "-lat" << stringify(latitude) << "-lon" << stringify(longitude) << "-e" << stringify(elevation) << "-tz" << TimeZone;
 
     zmanimproc->start("java", args);
 
